package.json

```diff
@@ -12,13 +12,18 @@
   "configSchema": {
     "executablePath": {
       "default": "haml-lint",
-      "description": "Path to haml-lint executable",
+      "description": "This is the absolute path to your `haml-lint` command. You may need to run `which haml-lint` or `rbenv which haml-lint` to find this.",
       "type": "string"
     },
     "globalHamlLintYmlFile": {
       "default": "",
       "description": "Full path to a global Haml lint file, if no other is found",
       "type": "string"
+    },
+    "suppressWarnings": {
```

Comment:
This could possibly be misconstrued to mean "Don't show me `warning` severity messages".

Maybe a better name would be `suppressFailures`? Is there any reason we don't just want to show all errors though? Generally that means something is wrong and the user isn't going to get any usable output otherwise.

Reply:
What about changing the message to "Supress non-critical HAML-Lint messages" or a better way, adding a excerpt message to the results and show notification for errors only?.

Reply:
Something like this

```
        const messages = [];

        if (output.exitCode !== 0) {
          if (output.stderr.toLowerCase().startsWith(warning)) {
            messages.push({
              severity: warning,
              excerpt: `HAML-Lint: ${output.stderr}`,
              location: {
                file: filePath,
                // first line of the file
                position: [[0, 0], [0, Infinity]],
              },
            });
          }
        } else {
          showErrorNotification('HAML-Lint: Unexpected error', output.stderr);
        }
```

Reply:
Adding it as a message looks great to me!

Reply:
Should we add the unexpected errors this way too? I mean, without notification.

Reply:
The way I see it is:

- If the message prevents other results from being returned it should be an Atom error notification
- If other results can still be obtained from the linter then it probably should just be converted into a message on the entire first line

Reply:
:+1: